mlflow/metrics/genai/prompts/v1.py

```diff
@@ -17,33 +17,37 @@
 }
 grading_system_prompt_template = PromptTemplate(
     """
-Please act as an impartial judge and evaluate the quality of the provided output which
-attempts to produce output for the provided input based on a provided information.
+Task:
+You are an impartial judge. You will be given an input that was sent to a machine
+learning model, and you will be given an output that the model produced. You
+could also be given additional information that was used by the model to generate the output.
 
-You'll be given a grading format below which you'll call for each provided information,
-input and provided output to submit your justification and score to compute the {name} of
-the output.
+Your task is to determine a numerical score called {name} based on the input and output.
+A definition of {name} and a grading rubric are provided below.
+You must use the grading rubric to determine your score. You must also justify your score.
+
+Examples could be included below for reference. Make sure to use them as references and to
+understand them before completing the task.
```

Comment:
Can we only include this text if examples are actually specified? "Examples could be included" with zero examples to follow seems like it's more likely to confuse the LLM

Reply:
I am not sure how that is possible to relate a text that is in between the prompt with the examples that is added below? 
We could move it together so I can only add the entire example text if there are examples. Thoughts?

Reply:
If we only construct the prompt once we have the `examples`, then we can do this, right? I think we should do that

Reply:
Can you explain more about what you mean by construct the prompt once? 
I am not sure how that helps to dynamically add a statement to the prompt based on the variables that are added or not? 🤔